audiomate/utils/textfile.py

```diff
@@ -129,7 +132,7 @@ def read_separated_lines_generator(path, separator=' ', max_columns=-1,
         keep_empty (bool): If True empty columns are returned as well.
     """
     if not os.path.isfile(path):
-        print('File doesnt exist or is no file: {}'.format(path))
+        logger.error('File doesnt exist or is no file: {}'.format(path))
```

Comment:
By using `format()`, string interpolation will be performed regardless of the message actually being logged or not. That are potentially wasted CPU cycles. The logging system supports the old printf-style (with `*args`) which does not have the problem:

```
logger.warning('Protocol problem: %s', 'connection reset', extra=d)
```